cumulus_library_opioid/vocab/template_sql/create_included_keywords.sql.jinja

```diff
@@ -0,0 +1,18 @@
+{#- This query is an implicit version of 'get every potential rule
+that was found that matched a keyword-based rule' -#}
+CREATE OR REPLACE VIEW opioid__{{ steward }}_included_keywords AS
+SELECT
+    r.rxcui1,
+    r.rxcui2,
+    r.tty1,
+    r.tty2,
+    r.rui,
+    r.rel,
+    r.rela,
+    r.str1,
+    r.str2,
+    r.keyword
+FROM opioid__{{ steward }}_potential_rules AS r
+WHERE
+    r.REL NOT IN ('RB', 'PAR')
+    AND length(r.keyword) >= 4
```

Comment:
Sorry, this struck me as odd here - we control the keywords, yeah? Why not just not add small ones to the list?

Reply:
Right now, there are none that are smaller than 4 chars. I'm not sure if the intent is to force keywords to be relatively large? or if it's just 'mapping this logic to the data we have? @comorbidity

Reply:
per offline discussion, added a comment about this for future reference